### PR TITLE
testing.json file filled with data

### PR DESCRIPTION
**Pull request by prabhattopi**

### CODE REVIEW

**Comment by prabhattopi** [comment]
this is database file of backend of testing

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Hey, thanks
Could please create a new subcategory and `json` file under `youtube` category, named `testing` and then add your resources in `database/youtube/testing.json`

Let us know for any kinda help.

**Comment by prabhattopi** [comment]
ok sure

**Review by rupali-codes — COMMENTED** [review]
Thanks, i can not see it in the site. 
Now you will have to modify `data.ts` and `index.ts` files under `database` folder. Add this `testign` subcategory just like other components. Let's know for any kinda help :)

**Comment by prabhattopi** [comment]
done
please review once

File: database/youtube/testing.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        "name": "SoftwaretestingbyMKT",
+        "description": "This channel is for everyone who wants to learn Manual testing and Automation Testing for FREE.The reason he have made this YouTube channel is to educate people who want to learn in easy language. I have a good theory and practical knowledge on testing.",
+        "url": "https://www.youtube.com/@SoftwaretestingbyMKT",
+        "category": "youtube",
+        "subcategory": "testing",
+        "language": "english"
+    },
+    {
+        "name": "Automation Bro-Dilpreet johal",
+        "description": "This channel is beginner friendly as well take you up to advanced level in automation with almost every everyframework",
```
Comment (rupali-codes) [inline database/youtube/testing.json:12]:
```suggestion
        "description": "This channel is beginner friendly as well take you up to advanced level in automation with almost every framework",
```

**Comment by prabhattopi** [comment]
please review once thanks if any help regarding the backend frontend or optimizing please give me a flow I will definitely do that on weekends thanks once again

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**